homeassistant/components/device_automation/toggle_entity.py

```diff
@@ -170,9 +168,9 @@ async def async_attach_trigger(
 
 async def _async_get_automations(
     hass: HomeAssistant, device_id: str, automation_templates: list[dict], domain: str
```

Comment:
Shouldn't we type `automation_templates` more specifically too? Otherwise, how do we know that `automations` is correctly typed below?
```suggestion
    hass: HomeAssistant,
    device_id: str,
    automation_templates: list[dict[str, str]],
    domain: str,
```

Reply:
It would be good to do that, yes, and I don't have _that_ strong opinions on whether to include that here or to do it in a followup PR. But it would make my life somewhat easier if we didn't extend the scope of this one, because I do have one for device conditions and another for device triggers queued up which I'll file when this one is applied. toggle_entity is the one file they all touch, and the changes in this one are required by the other patches -- requires a bit of juggling and keeps those blocked if this one starts to take some time and changes.

Reply:
We could just commit my suggestion. It won't take any extra time. :+1:

Reply:
Yeah, you're probably right, I was thinking about some other code there.